semcore/accordion/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.24.0] - 2024-02-15
+
+### Changed
+
+- Add ability to set 'l' icon size for `Accordion.Item.Chevron`
```

Comment:
Please replace "Changed" with "Added" and remove "Add" from start of the note (to make it all be like "Added ability to set `l` icon size...)